elasticdl/doc/tutorials/elasticdl_local.md

```diff
@@ -0,0 +1,112 @@
+# ElasticDL on local
+
+This document aims to give a simple example to show how to sumbmit deep learning jobs to a local kubernetes cluster in a local computer. It helps to understand the working process of ElasticDL.
+
+
+## Environment prepare
+
+Here, we use macbook as our expriment environment.
+
+
+1. Install docker
+
+```bash
+brew cask install docker
+```
+
+2. Install kunernetes
+
+```bash
+brew install kubectl
+brew cask install minikube
+brew install docker-machine-driver-hyperkit
```

Comment:
Should we just link to minikube installation documentation? (users may not want to use brew) Also users should just select their own driver (users may not want to use this driver). We can point to a reference to hyperkit if needed.

Reply:
Yes, I will remove it.

Reply:
Yes, I will remove it.